dnf/automatic/main.py

```diff
@@ -340,6 +344,11 @@ def main(args):
             base.do_transaction()
             emitters.notify_applied()
             emitters.commit()
+
+            if (conf.commands.reboot == 'when-changed' or
+               (conf.commands.reboot == 'when-needed' and base.reboot_needed())):
+                if os.waitstatus_to_exitcode(os.system(conf.commands.reboot_command)) != 0:
+                    return 1
```

Comment:
Does it make sense to log an error message here? Something along the lines "reboot command failed".

Reply:
Cool, yeah, just added that.

Reply:
Thank you!